nvflare/private/fed/utils/fed_utils.py

```diff
@@ -501,3 +501,19 @@ def get_scope_prop(scope_name: str, key: str) -> Any:
     check_str("key", key)
     data_bus = DataBus()
     return data_bus.get_data(_scope_prop_key(scope_name, key))
+
+
+def get_job_launcher(job_meta: dict, fl_ctx: FLContext) -> dict:
+    engine = fl_ctx.get_engine()
+
+    job_launcher_ctx = engine.new_context()
```

Comment:
Should use the pattern:
with engine.new_context() as job_launcher_ctx:
....

Reply:
Changed.